src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -279,5 +283,23 @@ public boolean delete() {
         // TODO: Add journey state deletion.
         return Persistence.monitoredTrips.removeById(this.id);
     }
+
+    /**
+     * @return the zone id from the trip 'from' location.
+     */
+    public ZoneId tripZoneId() {
+        Optional<ZoneId> fromZoneId = getZoneIdForCoordinates(from.lat, from.lon);
+        if (fromZoneId.isEmpty()) {
+            String message = String.format(
+                "Could not find coordinate's (lat=%.6f, lon=%.6f) timezone for monitored trip %s",
+                from.lat,
+                from.lon,
+                id
+            );
+            throw new RuntimeException(message);
+        } else {
+            return fromZoneId.get();
+        }
+    }
```

Comment:
Please replace this with the `getTimezoneForTargetLocation` method that I made in #75 (see [here](https://github.com/ibi-group/otp-middleware/pull/75/files#diff-a3ee245c312c932d083bd39feecf69f3R331)) so that we get the timezone from the proper location depending on whether the trip is arrive by or depart at.

Reply:
Fixed in https://github.com/ibi-group/otp-middleware/pull/77/commits/a2e1ec3403a6fa2f002dcfd1c59f89594a6ca3a8